flutter_searchbox_ui/example/ios/Podfile.lock

```diff
@@ -30,4 +30,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 7368163408c647b7eb699d0d788ba6718e18fb8d
 
-COCOAPODS: 1.14.0
```

Comment:
@mohdashraf010897 If there isn't a specific reason for the downgrade, we should leave this as is.